dangerfile.js

```diff
@@ -114,7 +111,7 @@ const unreleasedModified = includes(modifiedFiles, 'UNRELEASED.yaml');
 const packagesModified = fileChanges.some(
   filePath => filePath.startsWith('packages/') && !filePath.endsWith('.md'),
 );
-if (packagesModified && !unreleasedModified && !declaredTrivial) {
+if (packagesModified && !unreleasedModified) {
```

Comment:
So this is even noisier now?

Reply:
Have you ever seen a PR where somebody actually added `#trivial` to the title to suppress an error? What's the point of having it if it's not used?

Reply:
If it was a label, I'd probably use it.

but yeah I guess it's so rare that we'd probably only remember the option after we see the warning anyway

Reply:
Yeahhh - to clarify, I think it's a good idea but it's clearly not used as is, so I figured remove it and then we can explore a better solution.

Reply:
Where is even `declaredTrivial` declared? Call me crazy but I can't seem to find it in the original file 😆 so must have done nothing anyway?